NEWS.md

```diff
@@ -5,6 +5,7 @@
   * Feature: Add `audit_description()` function
   * Feature: Add `expect_secure()` for the {testthat} package
   * Feature: Use `~/.ossindex/.oss-index-config`, if it exists
+  * Feature: Add a pkgdown website. Thanks to @josiahParry for the nice CSS.
   * Add Josiah Perry as contributor
```

Comment:
Parry * ;)

Reply:
Sorry:(

Reply:
Sorry couldn't resist:

https://www.youtube.com/watch?v=8cuihrjLNAo

Reply:
loool! I sometimes say "Parry, like the verb." which isn't always understood.